clicommand/secret_get.go

```diff
@@ -29,9 +29,25 @@ type SecretGetConfig struct {
 }
 
 var SecretGetCommand = cli.Command{
-	Name:        "get",
-	Usage:       "Get a secret by its key",
-	Description: "Get a secret by key from Buildkite Pipelines Secrets and print it to stdout.",
+	Name:  "get",
+	Usage: "Get a secret by its key",
```

Comment:
maybe we should keep the `and print it to stdout` bit in the usage - otherwise it could be a bit unclear where the secret goes? ie is it crammed directly into env? (no, but the user wouldn't know that)

Reply:
This wasn't originally in the 'Usage' field. It was in the 'Description', which I've now expanded upon, and I've retained this piece of info.

TBH, I'm not sure where the 'Usage' field value is actually surfaced, but I'll add it nonetheless.

Reply:
`Usage` is surfaced in the parent command's help - ie, currently, calling `buildkite-agent secret --help` returns 
```
Usage:

  buildkite-agent secret <command> [options...]

Available commands are:

  get   Get a secret by its key
  help  Shows a list of commands or help for one command


Options:

  --help, -h  show help
```

Reply:
but now that you mention it, it's a relatively obscure place to be putting info

Reply:
Aha, of course! I'll take note of this for next time :-)